src/test/java/LoginPageTest.java

```diff
@@ -7,13 +7,10 @@
 import pages.LoginPage;
 import utils.WebDriverUtils;
 
+import static colors.ColorsHex.*;
+import static page.url.PageUrl.ACCOUNT_PAGE_URL;
```

Comment:
Here in setUp method should login page url used, instead of `ACCOUNT_PAGE_URL + "login/"`
See:
`driver.get(ACCOUNT_PAGE_URL + "login/")`;

Reply:
Also FORGOT_PASSWORD_PAGE_URL should be defined in PageUrl class and used in assertion:
`softAssertions.assertThat(loginPage.getForgotYourPasswordLink()).isEqualTo(ACCOUNT_PAGE_URL + "forgotpassword/");`

And for following assertion CREATE_ACCOUNT_PAGE should be used:
 `softAssertions.assertThat(loginPage.getCreateAccountButtonLink()).isEqualTo(ACCOUNT_PAGE_URL + "create/");`